pkg/controllers/user/logging/clusterlogging.go

```diff
@@ -67,21 +71,7 @@ func (c *ClusterLoggingSyncer) Sync(key string, obj *v3.ClusterLogging) error {
 		if err := utils.RemoveEmbeddedTarget(c.deployments, c.serviceAccounts, c.services, c.roles, c.rolebindings); err != nil {
 			return err
 		}
-
-		allDisabled, err := utils.IsAllLoggingDisable(c.clusterLoggings.Controller().Lister(), c.projectLoggingLister)
-		if err != nil {
-			return err
-		}
-
-		if allDisabled {
-			if err := utils.RemoveFluentd(c.daemonsets, c.serviceAccounts, c.clusterRoleBindings); err != nil {
-				return err
-			}
-			if err := utils.RemoveConfigMap(c.configmaps); err != nil {
-				return err
-			}
-		}
-		return nil
+		return cleanResource(c.daemonsets, c.serviceAccounts, c.configmaps, c.clusterRoleBindings, c.clusterLoggingLister, c.projectLoggingLister)
```

Comment:
What about making cleanResource a func of ClusterLoggingSyncer so that we do not have to pass so many parameters

Reply:
I don't do this because also the project logging use this func